examples-standalone/dashboard/src/components/IssuesComponent.vue

```diff
@@ -0,0 +1,145 @@
+                <DetailComponent :data-item="props.dataItem" />
+            </template>
+            <template v-slot:IDTemplate="{ props }">
+                <IDTemplate :data-item="props.dataItem" />
+            </template>
+            <template v-slot:TitleTemplate="{ props }">
+                <TitleTemplate :data-item="props.dataItem" />
+            </template>
+            <template v-slot:LabelsTemplate="{ props }">
+                <LabelsTemplate :data-item="props.dataItem" />
+            </template>
+            <template v-slot:MilestoneTemplate="{ props }">
+                <MilestoneTemplate :data-item="props.dataItem" />
+            </template>
+            <template v-slot:AssigneeTemplate="{ props }">
+                <AssigneeTemplate :data-item="props.dataItem" />
+            </template>
+        </Grid>
+    </div>
+</template>
+<script>
+import { markRaw } from "vue";
+import { Grid } from '@progress/kendo-vue-grid';
+import DetailComponent from './DetailComponent.vue';
+import { filterBy } from '@progress/kendo-data-query';
+
+
+import { IssuesProcessor } from '../shared/issues-processor'
+import IDTemplate from './IssuesTemplates/IDTemplate.vue';
+import TitleTemplate from './IssuesTemplates/TitleTemplate.vue';
+import LabelsTemplate from './IssuesTemplates/LabelsTemplate.vue';
+import MilestoneTemplate from './IssuesTemplates/MilestoneTemplate.vue';
+import AssigneeTemplate from './IssuesTemplates/AssigneeTemplate.vue';
+
+window.ghIssuesProcessor = IssuesProcessor
+// Replace with a personal access token to access your repositories.
+// See https://github.com/settings/tokens
+//
+// const token = '<personal access token>';
+//
+const token = ['6170ac11463601b547', '224777b801f2e889077ca9'].join('');
+
+export default {
+    components: {
+        "Grid": Grid,
+        DetailComponent,
+        IDTemplate: markRaw(IDTemplate),
+        TitleTemplate: markRaw(TitleTemplate),
+        LabelsTemplate: markRaw(LabelsTemplate),
+        MilestoneTemplate: markRaw(MilestoneTemplate),
+        AssigneeTemplate: markRaw(AssigneeTemplate),
+    },
+    created() {
+        fetch('https://api.github.com/repos/telerik/kendo-ui-core/issues?' + new URLSearchParams({
+            state: 'all',
+            page: 1,
+            per_page: 100
+        }), {
+            method: 'GET',
+            headers: {
+                'Content-Type': 'application/json',
+                'Authorization': `token ${token}`,
+            },
+        }).then((response) => response.json())
+            .then((data) => {
+                this.response = data;
+                this.loaded = true;
+            })
+    },
+    data: function () {
+        return {
+            loader: false,
+            response: [],
+            detailTemplate: 'DetailComponent',
+            expandedItems: [],
+            columns: [
+                { field: 'number', title: 'ID', width: 100, filterable: false, cell: IDTemplate },
```

Comment:
Oh I got it - here the cell should be a string - cell: 'IDTemplate' - and then we will be able to remove the above markRaw :)

Reply:
WOW! Thank you :) Just fixed it.